Classes/Event/EventPublisher.php

```diff
@@ -45,46 +47,65 @@ class EventPublisher
 
     /**
      * @param EventStore $eventStore
-     * @param EventBus $eventBus
+     * @param EventListenerLocator $eventListenerLocator
      * @param PropertyMapper $propertyMapper
      * @param EventTypeResolver $eventTypeResolver
      */
-    public function __construct(EventStore $eventStore, EventBus $eventBus, PropertyMapper $propertyMapper, EventTypeResolver $eventTypeResolver)
+    public function __construct(EventStore $eventStore, EventListenerLocator $eventListenerLocator, PropertyMapper $propertyMapper, EventTypeResolver $eventTypeResolver)
     {
         $this->eventStore = $eventStore;
-        $this->eventBus = $eventBus;
+        $this->eventListenerLocator = $eventListenerLocator;
         $this->propertyMapper = $propertyMapper;
         $this->eventTypeResolver = $eventTypeResolver;
     }
 
 
     /**
      * @param string $streamName
```

Comment:
Actually, this would be the time now to add a description for what this method actually does. It's not easy to spot right away (for me).